windows/src/desktop/kmshell/install/UfrmInstallKeyboardLanguage.pas

```diff
@@ -157,7 +162,18 @@ function InstallKeyboardLanguage(Owner: TForm; const KeyboardID, ISOCode: string
   end
   else
     CheckForMitigationWarningFor_Win10_1803(Silent, '');
-
+  // Enable the keyboard
+  n := kmcom.Keyboards.IndexOf(KeyboardID);
+  if n < 0 then
+  begin
+    // The Keyboard was successully installed for the BCP47Code
+    // for some reason the index look up has failed. Still pass through
+    // the DoInstall reasult, however the keyboard will not be enabled.
+    TKeymanSentryClient.Client.MessageEvent(Sentry.Client.SENTRY_LEVEL_ERROR, 'KeyboardID: '+KeyboardID+' not found');
```

Comment:
A bit more context is probably helpful for error reports -- and including 'InstallKeyboardLanguage' helps for filtering on Sentry.

```suggestion
    TKeymanSentryClient.Client.MessageEvent(Sentry.Client.SENTRY_LEVEL_ERROR, 'InstallKeyboardLanguage: KeyboardID "'+KeyboardID+'" not found, attempting to install for language "'+ISOCode+'".');
```